ros2bag/ros2bag/verb/record.py

```diff
@@ -91,8 +91,12 @@ def add_arguments(self, parser, cli_name):  # noqa: D102
         self._subparser = parser
 
     def main(self, *, args):  # noqa: D102
+        # both all and topics cannot be true
         if args.all and args.topics:
             return print_error('Invalid choice: Can not specify topics and -a at the same time.')
+        # both all and topics cannot be false
+        if not(args.all or (args.topics and len(args.topics) > 0)):
+            return self._subparser.print_help()
```

Comment:
Nit

```suggestion
            return print_error('Invalid choice: Must specify topic(s) or -a')
```

This way the error is clear instead of generally printing the help function.